### ISSUE TITLE
PCjs 80386 boot hangs

### ISSUE THREAD

**Issue by yuhong** [jeffpar/pcjs.v1#46]
Go to http://www.pcjs.org/disks/pcx86/windows/win95/4.00.950/debugger/ . Click Reset and then Run. Notice it hangs. Now, click Halt. Notice the ROM addresses now have zero data values.

**Comment by yuhong** [comment]
For example u f000:c914 on reset, then u f000:c914 when it hangs. Notice the different values.

**Comment by jeffpar** [comment]
Yes, it’s a probably a problem with Compaq’s ROM-in-write-protected-RAM feature (or rather, my emulation of it).

**Comment by yuhong** [comment]
Still a problem.
